cmd/vcert/commands.go

```diff
@@ -720,37 +719,27 @@ func generateCsrForCommandGenCsr(cf *commandFlags, privateKeyPass []byte) (priva
 	return
 }
 
-func writeOutKeyAndCsr(cf *commandFlags, key []byte, csr []byte) (err error) {
+func writeOutKeyAndCsr(commandName string, cf *commandFlags, key []byte, csr []byte) (err error) {
+	pcc := &certificate.PEMCollection{}
+	pcc.Certificate = string(csr[:])
```

Comment:
We should add "CSR" (omitempty) to the PEMCollection struct so that the data is accurately named when we output this in JSON.  Then this will become `pcc.CSR = string(csr[:])`.

Reply:
Added CSR attribute to PEMCollection{} object. And updated code to make use of it.